### ISSUE TITLE
Bug: When a nested window/menu/dialog opens, I cannot use Hurl until I restart

### ISSUE THREAD

**Issue by riotrah** [U-C-S/Hurl#116]
I'm not at my Windows PC rn, but when I get to it I'll take a screen recording.

For now, here's the steps:

- Click a URL in any app on windows
- Hurl appears (the browser picker window)
- Open one of the menus, eg, by clicking the settings button (in my case, I accidentally press one of the keyboard shortcuts)
- The menu opens
- Try to click _anywhere_ in the menu, even the close button, or in the rest of Hurl
- That menu, and the rest of Hurl (ie main the browser picker window) disappear.
- The URL initially clicked did not get opened.

Now, try to click the same URL again:
- Hurl appears (the browser picker window)
- But the menu from before is _also_ still open
- Try to click anything inside the menu, in the main hurl window, or any other app etc
- Hurl and all windows close
- The URL still did not get opened.

Basically, if I ever open _any_ menu/other Hurl window, Hurl gets stuck there until I quit and restart Hurl.

**Comment by U-C-S** [comment]
I tried to follow the steps, I can't reproduce the issue, even after trying various ways that I suspected. Occasionally the edit URL window's close button is breaking like button disappears in flashes but nothing as described as above.
